client/src/main/com/sinch/sdk/models/DualToneMultiFrequency.java

```diff
@@ -0,0 +1,98 @@
+package com.sinch.sdk.models;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/**
+ * DTMF played to the callee.
+ *
+ * <p>Valid characters in the string are "0"-"9", "#" and "w". A "w" will render a 500 ms pause.
+ *
+ * <p>Example: "ww1234#w#" will render a 1s pause, the DTMF tones "1", "2", "3", "4" and "#"
+ * followed by a 0.5s pause and finally the DTMF tone for "#".
+ *
+ * <p>This can be used if the callout destination for instance require a conference PIN code or an
+ * extension to be entered.
+ */
+public class DualToneMultiFrequency {
+
+  private static final Pattern PATTERN = Pattern.compile("([0-9#w])*");
+  private final String dtfm;
+
+  private static boolean strict = true;
+
+  /**
+   * Create an instance of DualToneMultiFrequency
+   *
+   * @param dtfm The dtfm value. Valid characters in the string are "0"-"9", "#" and "w"
+   * @throws IllegalArgumentException Throw an exception if value contains invalid characters
+   */
+  public DualToneMultiFrequency(String dtfm) throws IllegalArgumentException {
+
+    if (strict && !validate(dtfm)) {
+      throw new IllegalArgumentException(
+          String.format("Invalid DTFM format for '%s' number", dtfm));
+    }
+    this.dtfm = dtfm;
+  }
+
+  /**
+   * Get the DTFM value
+   *
+   * @return The dtfm value
+   */
+  public String stringValue() {
+    return dtfm;
+  }
+
+  /**
+   * Create an instance from String
+   *
+   * @param value See {@link DualToneMultiFrequency(String)} constructor
+   * @return a DualToneMultiFrequency
+   */
+  public static DualToneMultiFrequency valueOf(String value) {
+    return new DualToneMultiFrequency(value);
+  }
+
+  /**
+   * Configure if invalid string values will throw an Exception or not
+   *
+   * @param strict Set strict mode to true/false
+   */
+  public static void setStrict(boolean strict) {
```

Comment:
It seems this method is not thread safe 🙃

Reply:
Exact :-(
it is a static and global variable: aim is if end user is having issue this will have to be set a global level on "application" startup
But you're absolutely right: changed to be thread safe (also done for E164PhoneNumber class)